lxd/instance/drivers/driver_qemu.go

```diff
@@ -2167,7 +2167,7 @@ mount_virtiofs() {
 
 mount_9p() {
     /sbin/modprobe 9pnet_virtio >/dev/null 2>&1 || true
-    /bin/mount -t 9p config "${PREFIX}/.mnt" -o access=0,trans=virtio >/dev/null 2>&1
+    /bin/mount -t 9p config "${PREFIX}/.mnt" -o access=0,trans=virtio,size=1048576 >/dev/null 2>&1
```

Comment:
@stgraber @monstermunchkin does this change also need to be applied to distrobuilder?

Reply:
Yeah, we need to make the same change over there